webapp/crashstats/crashstats/views.py

```diff
@@ -131,9 +131,12 @@ def report_index(request, crash_id, default_context=None):
     if request.user.has_perm("crashstats.view_pii"):
         context["report"]["current_signature"] = generate_signature(context["report"])
 
-    context["product_details"] = productlib.get_product_by_name(
-        context["report"]["product"]
-    )
+    try:
+        context["product_details"] = productlib.get_product_by_name(
+            context["report"]["product"]
+        )
+    except productlib.ProductDoesNotExist:
+        context["product_details"] = {}
```

Comment:
This should have handled products that don't exist. We could have crash data and then decide to stop supporting a product--you should be able to see the report views.